server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -104,78 +104,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion V_8_14_0 = def(8_636_00_1);
     public static final TransportVersion V_8_15_0 = def(8_702_00_2);
     public static final TransportVersion V_8_15_2 = def(8_702_00_3);
-    public static final TransportVersion QUERY_RULES_LIST_INCLUDES_TYPES_BACKPORT_8_15 = def(8_702_00_4);
```

Comment:
@kderusso @markjhoy There are no more 8.15 releases planned, so this backport can just be removed here

Reply:
Thanks for cleaning that up!